package.json

```diff
@@ -29,38 +29,40 @@
   "homepage": "https://github.com/antfu/taze#readme",
   "funding": "https://github.com/sponsors/antfu",
   "dependencies": {
-    "@antfu/ni": "^0.5.6",
-    "chalk": "^4.1.0",
-    "cli-progress": "^3.9.0",
-    "debug": "^4.3.2",
+    "@antfu/ni": "^0.12.0",
+    "@typescript-eslint/parser": "5.10.0",
+    "chalk": "^5.0.0",
+    "cli-progress": "^3.10.0",
+    "debug": "^4.3.3",
     "deepmerge": "^4.2.2",
-    "execa": "^5.0.0",
-    "fast-glob": "^3.2.5",
-    "find-up": "^5.0.0",
+    "execa": "^6.0.0",
+    "fast-glob": "^3.2.11",
+    "find-up": "^6.2.0",
     "libnpmconfig": "^1.2.1",
-    "pacote": "^11.3.1",
+    "pacote": "^12.0.2",
+    "rollup": "2.64.0",
     "semver": "^7.3.5",
-    "yargs": "^16.2.0"
+    "yargs": "^17.3.1"
```

Comment:
why add extra dependencies?

Reply:
I did not, I just ran an up date on all outdated deps